nbs/examples/Automatic_Hyperparameter_Tuning.ipynb

```diff
@@ -22,9 +22,11 @@
     " 3. Sample configurations with a search algorithm, train models, and evaluate them on the validation set.\n",
```

Comment:
<p>I think it'd be good before this to show how to explore the results, like with ray above. I think <code>nf.results.trials_dataframe()</code> produces the same result.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/767/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>